src/functions/searches/subscribe-to-one-search/subscribe-to-one-search.ts

```diff
@@ -99,26 +115,51 @@ export const makeSubscribeToOneSearch = (context: APIContext) => {
 			map(msg => toSearchEntries(msg)),
 		);
 
-		const _filter$ = new BehaviorSubject(initialFilter);
+		const _filter$ = new BehaviorSubject<SearchFilter>(initialFilter);
 		const setFilter = (filter: SearchFilter) => {
 			_filter$.next(filter);
 		};
-		const filter$ = _filter$.asObservable().pipe(distinctUntilChanged((a, b) => isEqual(a, b)));
+		const filter$ = _filter$.asObservable().pipe(
+			startWith<SearchFilter>(initialFilter),
+			bufferCount(2, 1),
+			map(
+				([prev, curr]): RequiredSearchFilter => ({
+					entriesOffset: {
+						index: curr.entriesOffset?.index ?? prev.entriesOffset?.index ?? initialFilter.entriesOffset.index,
+						count: curr.entriesOffset?.count ?? prev.entriesOffset?.count ?? initialFilter.entriesOffset.count,
+					},
+					dateRange: {
+						start: curr.dateRange?.start ?? prev.dateRange?.start ?? initialFilter.dateRange.start,
+						end: curr.dateRange?.end ?? prev.dateRange?.end ?? initialFilter.dateRange.end,
+					},
+					desiredGranularity: curr.desiredGranularity ?? prev.desiredGranularity ?? initialFilter.desiredGranularity,
+					fieldFilters: curr.fieldFilters ?? prev.fieldFilters ?? initialFilter.fieldFilters,
+				}),
+			),
+			distinctUntilChanged((a, b) => isEqual(a, b)),
+		);
+
+		const requestEntries = async (filter: RequiredSearchFilter): Promise<void> => {
+			const first = filter.entriesOffset.index * filter.entriesOffset.count;
```

Comment:
Shouldn't this just be `const first = filter.entriesOffset.index;`?

If I wanted to refer to indexes 2 through 6 inclusive, I would expect to use `{ index: 2, count: 5 }`
```
          v index
| 0 | 1 | 2 | 3 | 4 | 5 | 6 | 7 |
        |<-    count      ->|
```

We should let the user do the pagination logic themselves.